swo/mpt/extensions/runtime/djapp/conf/default.py

```diff
@@ -173,9 +167,16 @@
     },
 }
 
+# Proxy settings
+USE_X_FORWARDED_HOST = True
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# MPT settings
+
 MPT_API_BASE_URL = os.getenv("MPT_API_BASE_URL", "http://localhost:8000")
 MPT_API_TOKEN = os.getenv("MPT_API_TOKEN", "change-me!")
-PRODUCT_ID = os.getenv("PRODUCT_ID", "PRD-1111-1111-1111")
+MPT_PRODUCT_ID = os.getenv("MPT_PRODUCT_ID", "PRD-1111-1111-1111")
```

Comment:
We should notify Sylwester about this changes

Reply:
Yeah you are right, I modified it in the TDRs but I forgot to mention it in the changes notification email.
I will create a teams group with Sylvester you and me to stay in sync until the first deployment succeed.